scripts/globals/spells/trust/curilla.lua

```diff
@@ -0,0 +1,26 @@
+-----------------------------------------
+-- Spell: Curilla
+-----------------------------------------
+	require("scripts/globals/trusts");
+-----------------------------------------
+
+function onMagicCastingCheck(caster,target,spell)
+    if (not caster:canUseMisc(dsp.zoneMisc.PET)) then
+        return dsp.msg.basic.CANT_BE_USED_IN_AREA
+    elseif (not caster:hasTrust(dsp.trust.CURILLA)) then
+		local msg = caster:checkTrust(dsp.trust.CURILLA);
+		if(msg > 0) then
+			return msg
+		end
+    else
+        return 299
+    end
+	return 0;
+end
+
+function onSpellCast(caster,target,spell)
+    -- Todo: put Trusts's spawn in dialog here or in entity script?
```

Comment:
hey lets not copy a comment made back at the begging of those into every damn script

Reply:
Yea sorry, i fixed this